common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -251,6 +254,32 @@ private ItemAnnotation calculateAnnotation(ItemStack itemStack, StyledText name)
         return annotation;
     }
 
+    private StyledText simplifyName(StyledText name) {
+        String str = name.getString();
+
+        final Pattern[] PATTERNS = {
+            Pattern.compile("^§6(?:Buying|Selling) [^ ]+ (.+?)(?:§6)? for .+ Each$"),
```

Comment:
This is knowledge that the ItemHandler should not really have.

I was going to suggest to have a look at `registerKnownMarkerNames` for how to do a dependency inversion, and move this to the TradeMarketModel. Then I realized we still do not have a TradeMarketModel, and maybe we should not create one in this PR...

I might be willing to accept this in ItemHandler for now, but as soon as we get the TradeMarketModel, it needs to go there instead.

Reply:
This is a good idea. Should I just leave a comment explaining that it's temporary?